templates/config.toml

```diff
@@ -35,16 +35,23 @@ oom_score = 0
     enable_tls_streaming = false
     max_container_log_line_size = 16384
     [plugins.cri.containerd]
-      snapshotter = "overlayfs"
```

Comment:
Are we sure we want to remove this line? Just verifying.

Reply:
Yeah, it's not in the recommended default config anymore (which is very transient).